components/MarkdownTextBlock/src/MarkdownTextBlock.xaml

```diff
@@ -0,0 +1,26 @@
+<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:labs="using:CommunityToolkit.Labs.WinUI.MarkdownTextBlock">
+
+    <Style BasedOn="{StaticResource DefaultMarkdownTextBlockStyle}"
+           TargetType="labs:MarkdownTextBlock" />
+
+    <Style x:Key="DefaultMarkdownTextBlockStyle"
+           TargetType="labs:MarkdownTextBlock">
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="labs:MarkdownTextBlock">
+                    <Grid Padding="{TemplateBinding Padding}"
+                          HorizontalAlignment="{TemplateBinding HorizontalAlignment}"
+                          VerticalAlignment="{TemplateBinding VerticalAlignment}"
+                          Background="{TemplateBinding Background}"
+                          BorderBrush="{TemplateBinding BorderBrush}"
+                          BorderThickness="{TemplateBinding BorderThickness}"
+                          CornerRadius="{TemplateBinding CornerRadius}">
+                        <Grid x:Name="MarkdownContainer" />
```

Comment:
Is there the need for the nested grids? Why can't the content go in the outer grid? Otherwise, could the inner one be a Border or ContentPresenter?

Reply:
Removed in the latest version